scrapers/sis_scraper/main.py

```diff
@@ -302,6 +298,8 @@ async def scrape_term(term):
     for school in schools:
         school["depts"] = sorted(school["depts"], key=itemgetter("code"))
 
+    os.makedirs(f"data/{term}", exist_ok=True)
```

Comment:
duplicate of line 255

Reply:
ah good catch. Looks like I messed up during my merge